llvm/lib/Target/AMDGPU/AMDGPULegalizerInfo.cpp

```diff
@@ -4204,6 +4204,8 @@ bool AMDGPULegalizerInfo::loadInputValue(Register DstReg, MachineIRBuilder &B,
     // TODO: Should we try to emit this once in the entry block?
     const LLT S32 = LLT::scalar(32);
     const unsigned Mask = Arg->getMask();
+    // None of the target SGPRs or VGPRs are expected to have a 'zero' mask.
+    assert(Mask && "Invalid mask.");
```

Comment:
Could you hide this in getMask

Reply:
Done @arsenm. Thanks for explaining as well!